### ISSUE TITLE
Can't load install/index.php

### ISSUE THREAD

**Issue by Kzenn** [craigk5n/webcalendar#131]
I cannot complete the install on a debian 9.5 with  apache2, php 7 and mysql 5.6. I am trying to load the install/index.php to complete the setup and it loads a blank page.
When I convert the page to HTML it's still blank
but I can load files from the WebCalendar repository

I would appreciate any help

**Comment by FranzGotsis** [comment]
What version of WebCalendar are you running?
First thing to check is whether the file is actually at the location you are looking for (FTP-Connect, Provider Tool, SSH, ..).

If the file is there (and the WWW-Server allows the access) you should see at least something.
Another test could be to replace index.php (make a backup first) with a simple html-Hello World file, then a PHP hello world file. 
For instance: 
<?php
	phpinfo();
?>

and check whether that works.
If this test file works there must be a PHP problem and you should go and check the PHP error logs.

In case you need more help just answer to this post.
kind regards
Franz Gotsis from Munich

**Comment by polymathuk** [comment]
I had this problem with a state-of-the-art debian 11 LAMP under LXC.
The problem was caused by deprecated `@set_magic_quotes_runtime ($magic);` function calls in multiple places in the install/index.php file. There were about 5 occurrences that I commented out using // at the beginning of the lines. This fixed it. It's an easy debug by watching
`tail -f /var/log/apache2/error.log`
fixing each line and refreshing the web browser page until it succeeds.
